Sound effects toggle

nicolasdellatorre opened the pull request:
In the game option screen and the end round screen a sound effects toggle has been implemented. Before playing every sound effect, it checks if the sound effects are activated otherwise no sound effect will be played.

adolfintel commented on CiarnuroRT/webapp/index.html:
```diff
@@ -39,6 +39,13 @@
                         <input type="submit" tabIndex="4" style="display:none" />
                     </form>
                 </div>
+                <div style="position:absolute;bottom:1rem;right:2rem;line-height:2rem">
```
Don't do this, it can mess up the layout if the screen is small or the font size is large. Just put it in the form with the rest of the settings.

adolfintel commented on CiarnuroRT/webapp/style.css:
```diff
@@ -240,6 +240,54 @@ input[type="date"]{
+.slider:before {
+  position: absolute;
+  content: "";
+  height: 26px;
```
Please don't use px in modern CSS, convert it to rem

adolfintel commented on CiarnuroRT/webapp/ciarnurort.js:
```diff
@@ -134,6 +135,16 @@ function initSoundSystem(){
 }
+function soundFxToggle(){
+    soundFxOn=!soundFxOn;
+    if(soundFxOn){
```
You can just use `I("initialSoundToggle").checked=I("endroundSoundToggle").checked=soundFxOn;`

nicolasdellatorre commented:
I have fixed the layout by inserting the checkbox inside the form, I have converted the pixel measures into rem and I have simplified the javascript sound effects toggle function.
